Dockerfile

```diff
@@ -14,7 +14,8 @@ COPY --chown=$username suzieq/config/etc/suzieq-cfg.yml /home/$username/.suzieq/
 
 RUN mkdir /home/$username/parquet && \
     chown $user_id:$user_id /home/$username/parquet && \
-    sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml
+    sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml && \
+    sed -i 's/# log-stdout:/log-stdout:/' /home/$username/.suzieq/suzieq-cfg.yml
```

Comment:
Why don't we just fix the configuration file?

Reply:
Because that'd be so easy, and why would we want that :(. Stupid me. Fixing.